src/components/FormSubmit/index.tsx

```diff
@@ -55,15 +61,15 @@ function FormSubmit({innerRef, children, onSubmit, style}) {
 
             form.removeEventListener('submit', preventDefaultFormBehavior);
         };
-    }, [innerRef]);
+    }, [ref]);
 
     return (
         // React-native-web prevents event bubbling on TextInput for key presses
         // https://github.com/necolas/react-native-web/blob/fa47f80d34ee6cde2536b2a2241e326f84b633c4/packages/react-native-web/src/exports/TextInput/index.js#L272
         // Thus use capture phase.
         <View
-            role={ComponentUtils.ACCESSIBILITY_ROLE_FORM}
-            ref={innerRef}
+            accessibilityRole={ComponentUtils.ACCESSIBILITY_ROLE_FORM as AccessibilityRole}
```

Comment:
why you are changing `role` to `accessibilityRole` ? for RN Web we need to pass `role` 
https://necolas.github.io/react-native-web/docs/accessibility/

Reply:
It was changed by mistake

Reply:
reverted